hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoop/HadoopPlugin.groovy

```diff
@@ -94,7 +94,9 @@ class HadoopPlugin implements Plugin<Project> {
 
       // Add clusterProvided jars on the classpath, this will not add transitive dependencies of each jar
       project.sourceSets.main.compileClasspath += clusterProvided
-      project.javadoc.classpath += clusterProvided
+      if (project.javadoc.hasProperty("classpath")) {
```

Comment:
This code is inside a project.getPlugins().withType(JavaPlugin) { ... } block, so it is safe to assume the javadoc task is present on the project. @pranayhasan instead of project.javadoc (which will indeed find the task), let's indeed follow the convention of using either project.tasks.javadoc or project.tasks['javadoc'], which makes it very explicit that you are looking up the task and not some other kind of extension property.